### ISSUE TITLE
Can't reproduce reported results.

### ISSUE THREAD

**Issue by OreoChocolate** [mf-zhang/Optical-Flow-in-the-Dark#1]
Hi, thanks for the great work!

![image](https://user-images.githubusercontent.com/19465114/115876976-e97dbc00-a481-11eb-947e-a773a4e236ac.png)
I run your code on VBOF dataset using FlownetS pretrained model. It should produce 5.46 ~ 6.26 epe shown in the paper.

but I got the over 25 epe on every brightness setting. 

Could you please tell me where I'm wrong? 
Thanks.

**Comment by mf-zhang** [comment]
Thank you for your interest in our work!

You can try to visualize the optical flow results. If you can get reasonable visual result, the problem could be easily fixed when calculating EPE. 

If the visual result is still not reasonable, you can try to test the FlowNetS pretrained model published by its author, and re-train the FlowNetS with FlyingChairs dataset + image noise, which could achieve similar results as the paper showed.

**Comment by OreoChocolate** [comment]
@mf-zhang Thank you for replying. I try to re-train the FlowNets with FCDN dataset and I got 2.302 EPE on FCDN validation set. But I have the same issue when I test re-trained model on VBOF dataset (over 20 EPE).

I have no idea how can I fixed It.

when I visualized the optical flow results, It seems reasonable. How could I fix this problem?

I have other questions.
1. In the paper brightness levels are categorized as 7 levels (0~6). But In VBOF dataset, It has 14 levels (1~14). How could I categorize the brightness level like paper? Is there any criterion?

2.  Does "FlyingChairs dataset + image noise" mean that Flying Chairs dataset + FCDN or only FCDN?

Thanks in advance!

**Comment by dongliangcao** [comment]
> Hi, thanks for the great work!
> 
> ![image](https://user-images.githubusercontent.com/19465114/115876976-e97dbc00-a481-11eb-947e-a773a4e236ac.png)
> I run your code on VBOF dataset using FlownetS pretrained model. It should produce 5.46 ~ 6.26 epe shown in the paper.
> 
> but I got the over 25 epe on every brightness setting.
> 
> Could you please tell me where I'm wrong?
> Thanks.

I also cannot reproduce the results in VBOF dataset by using pre-trained PWC-Net both for pre-trained in FlyingChairs dataset and in FCDN dataset.

**Comment by a673660099** [comment]
Hello!

I can't reproduce the results in the VBOF dataset by using PWC Net for pre training in the FlyingChairs dataset and the FCDN dataset

Have obtained huge EPE（>20）.

**Comment by a673660099** [comment]
> 嗨，感谢您的出色工作！
> 
> ![图片](https://user-images.githubusercontent.com/19465114/115876976-e97dbc00-a481-11eb-947e-a773a4e236ac.png) 我使用 FlownetS 预训练模型在 VBOF 数据集上运行您的代码。它应该产生论文中显示的5.46~6.26 epe。
> 
> 但我在每个亮度设置上都获得了超过 25 个 epe。
> 
> 你能告诉我我哪里错了吗？ 谢谢。

Hello, have you solved the problem of EPE? We hope to inform you!!!!!!!

**Comment by a673660099** [comment]
> 感谢您对我们的工作感兴趣！
> 
> 您可以尝试可视化光流结果。如果能得到合理的视觉结果，在计算 EPE 时问题就很容易解决。
> 
> 如果视觉结果仍然不合理，可以尝试测试作者发布的 FlowNetS 预训练模型，并使用 FlyingChairs 数据集 + 图像噪声重新训练 FlowNetS，可以达到与论文相似的结果。

Hello, the visual result of optical flow is good, but the value of EPE is too large, so there is no problem in the calculation of EPE. Can you tell me how to calculate the indicators in the paper?
